lib/index.js

```diff
@@ -90,8 +94,10 @@ function load(options) {
 
     const humanConnectIFrame = document.createElement("iframe");
     const humanConnectModalOverlay = document.createElement("div");
+    // No token required when mode === select and preseed === false
+    const tokenRequired = options.mode !== "select" && options.preseed;
```

Comment:
if `mode` is equals to `select`, it does not consider the `preseed`value.... is this behavior expected? (per comment above)

Reply:
^ Indeed, this should be an OR.

Reply:
Yes! My bad, this should be `||`